src/python/build_management/build_manager.py

```diff
@@ -739,19 +744,17 @@ class FuchsiaBuild(RegularBuild):
   FUCHSIA_BUILD_REL_PATH = os.path.join('build', 'out', 'default')
   FUCHSIA_DIR_REL_PATH = 'build'
 
-  def __init__(self, *args, **kwargs):
-    super().__init__(*args, **kwargs)
-
-    # Decide per-build whether to use undercoat, based on rollout level
-    rollout_level = environment.get_value('FUCHSIA_UNDERCOAT_ROLLOUT_LEVEL', 0)
-    self.use_undercoat = random.random() < rollout_level
-    logs.log('Undercoat enabled: %s' % self.use_undercoat)
+  def _pick_fuzz_target(self, fuzz_targets, target_weights):
+    """No-op, since Fuchsia builds pick targets later than other build types
+    and we aren't ready at the point that this is called by the superclass's
+    setup()."""
 
   def _get_fuzz_targets_from_dir(self, build_dir):
-    """Overridden to get targets list from fuchsia."""
+    """Returns a list of fuzz targets in a legacy Fuchsia build. For
+    undercoat-driven Fuchsia builds, a running instance is required to
+    enumerate targets so this is a no-op."""
```

Comment:
Can we raise an exception here now if `use_undercoat` is true, since this no now never be called in that case?

Reply:
Unfortunately this is still called by `RegularBuild.setup()` via `Build._unpack_build()`, so at least as things are now I think we don't want to throw an error in this case.